<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>173</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>173</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>173</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is a Filtering Window.&lt;/p&gt;&lt;p&gt;The Element that have the setted details will be shown.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>371</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>46</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Element</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="elem_andor_prop_cBox">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>or</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>and</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>99</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Element Property</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>89</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string> Property Detail</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="prop_andor_det_cBox">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>20</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>or</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>and</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="elemText">
      <property name="maximumSize">
       <size>
        <width>264</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="propText">
      <property name="maximumSize">
       <size>
        <width>264</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="detText">
      <property name="maximumSize">
       <size>
        <width>264</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
